CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.2.0
+
+- Add support for `limitBroadCastToOrigin` as a framebus config option
```

Comment:
one nit, `broadcast` is all one word, so shouldn't this be `limitBroadcastToOrigin`?

Reply:
oh yeah thank you, not sure when that crept in. I'll fix here and there other places.